lib/dirac_mobius.cpp

```diff
@@ -383,7 +387,7 @@ namespace quda {
     }
 
     // we desire solution to full system
-    auto tmp = getFieldTmp(b[0].Even());
+    auto tmp = getFieldTmp(x[0].Even());
```

Comment:
Similar question to 4-d preconditioned DWF, should this be multi-rhs-ified?

Reply:
And like dwf-4d: this will be in a followup PR...